core-bundle/contao/languages/en/default.xlf

```diff
@@ -284,6 +284,9 @@
       <trans-unit id="ERR.personalDataPassword">
         <source>There is at least one "personal data" module that allows the password to be changed. This is a security weakness and will no longer work in Contao 6. Use the "change password" module instead.</source>
       </trans-unit>
+      <trans-unit id="ERR.copyAllUnique">
+        <source>Could not copy record ID %d because it already exists.</source>
```

Comment:
This wording can be misleading I think as it is not obivous which field has caused the unique constraint to fail.

```suggestion
        <source>Could not copy record ID %d because its data conflicts with an existing record.</source>
```